smartsim/_core/control/manifest.py

```diff
@@ -187,7 +185,7 @@ def has_db_scripts(entity: t.Union[EntityList, SmartSimEntity]) -> bool:
         # we update has_db_objects so that as soon as one check
         # returns True, we can exit
         has_db_objects |= any(
-            [has_db_models(model) | has_db_scripts(model) for model in self.models]
+            has_db_models(model) | has_db_scripts(model) for model in self.models
```

Comment:
From the code above, these should always be `True` now?

Reply:
There's a difference between having the property and having items in the collection. If the underlying collections are empty, false will still be possible.